src/Symfony/Component/HttpKernel/Tests/Fixtures/BundleDependencies/BundleCDependenciesBA.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies;
+
+use Symfony\Component\HttpKernel\Bundle\BundleDependenciesInterface;
+
+class BundleCDependenciesBA implements BundleDependenciesInterface
+{
+    public function getBundleDependencies()
+    {
+        return array(
+            'Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies\BundleBDependenciesA',
```

Comment:
What about
```php
class AcmeBundle extends Bundle implements BundleDependenciesInterface
{
    public function getBundleDependencies()
    {
        return array(
            self::REQUIRED => array(
                // All values must be FQN strings to avoid bundles being loaded several times
                'FOS\HttpCacheBundle\FOSHttpCacheBundle',
                // if on php 5.5 and higher
                Oneup\FlysystemBundle\OneupFlysystemBundle::CLASS,
            ),
            self::OPTIONAL => array(
                // Optional dependencies can not use ::CLASS constant
                'FOS\HttpCacheBundle\FOSHttpCacheBundle',
        );
    }
}
 ```